Dockerfile

```diff
@@ -1,5 +1,5 @@
 #############      builder                                  #############
-FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
```

Comment:
The base image can also be updated to 
`eu.gcr.io/gardener-project/3rd/alpine:3.13.7` as done in [this](https://github.com/gardener/machine-controller-manager-provider-openstack/pull/44) PR

Reply:
Please update the image to `3.15.0` , as `3.13.7` doesn't exist